tests/Detector.Tests/Detector.Tests.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
To make sure that your tests are running in CI, for now, you can update the file called `build\testBuildScriptGenerator.sh` with the following content:

```
testProjectName="Detector.Tests"
cd "$TESTS_SRC_DIR/$testProjectName"
dotnet test \
    --blame \
    --test-adapter-path:. \
    --logger:"xunit;LogFilePath=$ARTIFACTS_DIR\testResults\\$testProjectName.xml" \
    -c $BUILD_CONFIGURATION
```